### DISCUSSION TITLE
Refine working agreement

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/methods#62] [Shaping]
Build on the outline we started in cycle 1 to further discuss and capture the working conventions for this product team. Continue work and discussion started in #49.

**Comment by pglevy** [comment]
@sofya-UX and @allie-shaw , I'm re-reading Lean UX, and they mention this as an example of a [working agreement](https://medium.com/@giffco/getting-tactical-product-team-working-agreements-b26aa46255ba) they like. ([Template](https://docs.google.com/spreadsheets/d/1EYkSwnKVh_Y9207c_2FYSILXA6U-vUVq73hHmIe-yT0/edit#gid=0)). Maybe this could work as an outline to build on.

**Comment by sofya-UX** [comment]
Reviewing the materials that @pglevy shared today. I spun up a version of the template and started customizing it and also capturing questions as I go. This draft can be accessed in Methods Teams' files [here](https://teams.microsoft.com/_#/files/General?threadId=19%3Af72f4f0d5d8844b198d7f9bc2c6abf53%40thread.tacv2&ctx=channel&context=Logistics&rootfolder=%252Fsites%252FMethods%252FShared%2520Documents%252FGeneral%252FLogistics)

**Comment by pglevy** [comment]
Some other references:
https://tech.gsa.gov/guides/agile_team_working_agreement/
https://34slpa7u66f159hfp1fhl9aur1-wpengine.netdna-ssl.com/wp-content/uploads/2018/10/Working-Agreement-Canvas-v1.24.pdf
https://derisking-guide.18f.gov/qasp/

**Comment by pglevy** [comment]
Started on a new outline and filling in some content last week and forgot push. It's still in progress but [up on the wiki](https://github.com/Bixal/methods/wiki/Bixal-Methods-team-working-agreement-(draft)) now. @Bixal/methods

**Comment by pglevy** [comment]
Capturing the outline from initial outline here:


# Previous content 👇

As members of Bixal UX team, we will be expected to dedicate an average of # hours a week to work towards the following goals:

## Goal 1: Define the way we work
* Together define the measurable objective for what we want to accomplish within the next 12 months
* Establish the internal review and contribution process that we will follow
* Establish the ceremonies that the team will perform (planning sessions, retros, etc.)
* Define how we will measure success of this effort

## Goal 2: Define the Methods
* Prioritize methods and collaboratively create a roadmap for the next 12 months
* Collectively agree on the definition of done

## Goal 2: Maintain the Methods
* Define the approach and criteria for refining Methods based on real-life execution and feedback from UX team members
* Refine and update Methods according to current practices

## Goal 3: Encourage adoption of the Methods
* Define the approach for Methods to be shared and educated upon outside the UX team
* Define collaboration touch points with teams outside UX team
* Determine the ways the UX team can support Methods learning and practicing across Bixal
* * Idea: Conduct research around the use of the Methods in order to increase usability and adoption.
